packages/plugin-connector/src/data-sources/exec-source.js

```diff
@@ -51,8 +51,7 @@ export const execSource = async (source, supportedDbs, outDir) => {
 				});
 			} else result = _r;
 		} catch (e) {
-			if (e instanceof z.ZodError)
-				console.log(/**@type {Error & { format: () => any}}*/ (e).format());
+			if (e instanceof z.ZodError) console.log(e.format());
```

Comment:
We might want to look at `clean-zod-errors.js` for this; but not a huge deal either way